### ISSUE TITLE
replace expiring dod-eca trusted certificate

### ISSUE THREAD

**Issue by wyattwalter** [department-of-veterans-affairs/va.gov-team#4185]
## User Story or Problem Statement

The SSL cert monitoring process is flagging the dod-eca.pem certificate that we trust on all nodes as expiring next month. I'm not sure what all uses this certificate, we'll need to unravel that a bit and see if we can find the renewed version of it if it's definitely needed.

## Goal
Evaluate usage of this certificate to determine if we still need it. If it's needed, find the new version of it that's not expiring.

## Objectives or Key Results this is meant to further
- Keep the site up

## Acceptance Criteria
- [ ] Replace the certificate if necessary, delete if it's not used

## How to configure this issue
- [x] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by wyattwalter** [comment]
@ricetj same exact question as https://github.com/department-of-veterans-affairs/va.gov-team/issues/4183

**Comment by wyattwalter** [comment]
The best hint I've got so far is this change where the dod-eca file was created.. renamed from being an mvi trusted chain. Going to see if I can see if there's any partners currently using this intermediate cert.

**Comment by wyattwalter** [comment]
I've spent a good deal of time trying to figure out if and how this cert was ever used, but haven't really come up with a lot so far.

In the git history, it looks like it was included in a commit while setting up MVI initially. But, it was included in a ball of certs at once. I've checked the chains of all the certs we have in our repo as well as the chains of everything we connect to from the fwdproxy. The only reference I can find is that MVI is sending back an `Acceptable client certificate CA names` list which includes this CA cert. However, the client cert we use wasn't signed by this certificate.

I've been unable to find any real references where this is actually in use. Once the release freeze and wave of updates passes, I'd like to try removing this CA cert from each environment at at time to check for any breakage.. but I _think_ we can safely just remove it.
